### PR TITLE
Make addon easier to run

### PR DESCRIPTION
**Pull request by ianb**
* Move the addon code to addon/
* Add package.json for dev dependencies
* Add 'npm run addon' to start a browser with the addon installed

### CODE REVIEW

**Comment by ianb** [comment]
jpm scans the directory that holds the add-on, so moving the add-on to a subdirectory seemed necessary (otherwise the profile, node_modules, etc would get scanned)

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "description": "A tab-splitting extension for Firefox",
+  "version": "0.1.0",
+  "author": "Mozilla (https://mozilla.org/)",
+  "bugs": {
+    "url": "https://github.com/mozilla/tab-split/issues"
+  },
+  "dependencies": {
+  },
+  "devDependencies": {
+    "jpm": "^1.3.1"
```
Comment (pdehaan) [inline package.json:12]: Should we be using [**jpm**](https://www.npmjs.com/package/jpm) or [**web-ext**](https://www.npmjs.com/package/web-ext)?

Comment (ianb) [reply package.json:12]: It's a bootstrap extension, so only jpm can run it. It'll break eventually, but it works for now!

**Comment by fzzzy** [comment]
Nice.

**Pull request merged**